snapd-glib/snapd-snap.c

```diff
@@ -1560,10 +1685,10 @@ snapd_snap_class_init (SnapdSnapClass *klass)
                                                           G_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
     g_object_class_install_property (gobject_class,
                                      PROP_WEBSITE,
-                                     g_param_spec_string ("website",
```

Comment:
Add a new property instead of changing the existing one - changing it breaks API.